routes/book.route.js

```diff
@@ -20,10 +21,13 @@ Router.route('/add')
         bookController.addBook
     );
 
-Router.route('/:id')
+Router.route('/:bookId')
     .get(isLoggedin(), userAuth('all'), bookController.renderBook)
     .delete(isLoggedin(), userAuth('all'), bookController.deleteBook);
```

Comment:
@saggy2001  You have renamed the `id` to `bookId`, you changed the `addBook` function also but didn't change in `deleteBook` function in `book.controller.js`. Please fix that.

Reply:
Okk I will do that

Reply:
Okay issue fixed.